analytics/tests/metrics/test_burndown.py

```diff
@@ -85,6 +85,41 @@ def test_count_tix_created_before_sprint_start(self):
         ]
         assert df.to_dict("records") == expected
 
+    def test_count_tix_closed_after_sprint_start(self):
```

Comment:
I'm very grateful for your dedication to writing and updating these tests! It will save so much pain later.